mappings/net/minecraft/client/search/IdentifierSearch.mapping

```diff
@@ -1,9 +1,9 @@
 CLASS net/minecraft/unmapped/C_ttylwehh net/minecraft/client/search/IdentifierSearch
-	FIELD f_ducunurc searchTree Lnet/minecraft/unmapped/C_mkylkzob;
+	FIELD f_ducunurc identifierSearcher Lnet/minecraft/unmapped/C_mkylkzob;
 	FIELD f_pjexfxok comparator Ljava/util/Comparator;
 	METHOD <init> (Ljava/util/function/Function;Ljava/util/List;)V
 		ARG 1 identifiers
-		ARG 2 values
+		ARG 2 searchables
```

Comment:
This is not a list of things, each of which is searchable; it's a single list (which is seearchable), so it shouldn't be plural.
Calling it `searchable` would technically make sense, but I prefer the existing `values` name (which also gets to follow the convention of making collections plural).

Reply:
I think you make a good point in that if it is confusing then a better name should be used, like values.

Just to describe how I was thinking about it though: each element in the list maps with a function to a stream of strings I believe.  So like if the searchables are ItemStacks, then each stack is searchable in that its name/lines of tooltip/whatever it is, I don't actually know if that's exactly what it represents in the case of ItemStacks or not.  So I was thinking both that the item stacks are searched but also that each ItemStack has its corresponding strings searched.